skimage/feature/texture.py

```diff
@@ -180,6 +180,10 @@ def graycoprops(P, prop='contrast'):
     - 'correlation':
         .. math:: \\sum_{i,j=0}^{levels-1} P_{i,j}\\left[\\frac{(i-\\mu_i) \\
                   (j-\\mu_j)}{\\sqrt{(\\sigma_i^2)(\\sigma_j^2)}}\\right]
+    - 'mean': :math:`\\sum_{i,j=0}^{levels-1} i*P_{i,j}`
+    - 'variance': :math:`\\sum_{i,j=0}^{levels-1} P_{i,j}*(i-mean)^2`
```

Comment:
The implementation looks fine, but I don't think these equations can be correct; they're missing the j term.

I suspect this is a histogram operation, in which case it should be:

```
:math:`\\sum_{i=0}^{levels-1} i*P_{i}`
```